src/groovy/org/pih/warehouse/order/OrderSummaryStatus.groovy

```diff
@@ -11,18 +11,22 @@ package org.pih.warehouse.order
 
 enum OrderSummaryStatus {
 
-    PENDING,
-    APPROVED,
-    PLACED,
-    PARTIALLY_SHIPPED,
-    SHIPPED,
-    PARTIALLY_RECEIVED,
-    RECEIVED,
-    PARTIALLY_INVOICED,
-    INVOICED,
-    COMPLETED,
-    CANCELED,
-    REJECTED
+    PENDING('warning'),
+    APPROVED('primary'),
+    PLACED('primary'),
+    PARTIALLY_SHIPPED('primary'),
+    SHIPPED('primary'),
+    PARTIALLY_RECEIVED('primary'),
+    RECEIVED('primary'),
+    PARTIALLY_INVOICED('primary'),
+    INVOICED('primary'),
+    COMPLETED('success'),
+    CANCELED('danger'),
+    REJECTED('danger')
+
```

Comment:
as commented above, added variants of circle's color for each status which will be the easiest to change in the future if added here in my opinion, so we do not have to provide any changes on the frontend side.